Parsers/ASimAuthentication/Tests/GoogleWorkspace_ASimAuthentication_SchemaTest.csv

```diff
@@ -0,0 +1,94 @@
+﻿Result
+"(0) Error: Missing mandatory field [Type]"
```

Comment:
@vakohl can you please fix these errors.

Reply:
@anki-narravula not seeing this error after re-running tests. Updated results in PR